src/main/resources/db/migration/V1_0__create_tables.sql

```diff
@@ -10,7 +10,9 @@ CREATE TABLE visit
     visit_start     timestamp with time zone NOT NULL,
     visit_end       timestamp with time zone NOT NULL,
     status          VARCHAR(80)   NOT NULL,
-    reasonable_adjustments    text
+    reasonable_adjustments    text,
+    create_timestamp timestamp default current_timestamp,
+    modify_timestamp timestamp default current_timestamp
```

Comment:
worth adding an index?

Reply:
Good point.  I will create a backlog item to review the existing schema.